src/main/java/org/ozeki/sms/dao/OzekiMessageOutDAO.java

```diff
@@ -1,10 +1,10 @@
 package org.ozeki.sms.dao;
-
 import org.ozeki.sms.valueholder.OzekiMessageOut;
-import org.springframework.data.repository.CrudRepository;
+//import org.springframework.data.repository.CrudRepository;
```

Comment:
Was this part of the ticket ? Seems to me like there couple of changes introduced not related to the ticket.

Reply:
Hello @sherrif10, you. are right, that was not part of the ticket. What happened is this, I had changes in the previous branch and those changes are still coming  newly created branch related to GSoC documentation

Reply:
And also @sherrif10 , I think that won't result into any problem

Reply:
You are right @sherrif10, that was not part of the ticket.   What happened is this, I had changes in the previous branch and those changes are still coming newly created branch related to GSoC documentation.

But @sherrif10 I think that won't result in any problem as long as I don't make changes to that.